src/app/datagrid/datagrid-angular-editor.demo.ts

```diff
@@ -304,16 +304,18 @@ export class DataGridAngularEditorDemoComponent implements AfterViewInit {
   constructor() { }
 
   ngAfterViewInit(): void {
-
-    this.sohoDataGridComponent.gridOptions = {
-      columns: EDITORS_COLUMNS,
-      dataset: EDITORS_DATA,
-      selectable: 'single',
-      idProperty: 'productId',
-      stretchColumn: 'last',
-      editable: true,
-      isList: true,
-      filterable: true
-    };
+    // in ngAfterViewInit/Checked must set inputs from a timeout so that change detection is executed.
+    setTimeout(() => {
```

Comment:
So i guess this is a breaking change. Is there any way to not have to do this?

Reply:
I don't think so, most folks are already doing this probably. It's surprising it worked at all w/o a timeout setting an input when in ngAfterviewInit and checked.